xml/System.Runtime.InteropServices.Marshalling/Utf16StringMarshaller.xml

```diff
@@ -70,8 +79,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Frees native resources.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+<xref:System.Runtime.InteropServices.Marshalling.CustomTypeMarshallerFeatures.UnmanagedResources>
```

Comment:
Phantom xrefs in this file.

Reply:
Fixed https://github.com/dotnet/dotnet-api-docs/pull/8185/files#r914244652